var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -40,7 +40,7 @@ class Elfutils(AutotoolsPackage):
     list_depth = 2
 
     version('0.168','52adfa40758d0d39e5d5c57689bf38d6')
-    version('0.163','77ce87f259987d2e54e4d87b86cbee41')
+    version('0.163','77ce87f259987d2e54e4d87b86cbee41', preferred=True)
```

Comment:
It should probably be a separate PR. Livdwarf doesn't build properly with the latest elfutils due to header issues.  This makes the build succeed in the tests.